src/dialogs/more-info/controls/more-info-light.js

```diff
@@ -282,6 +299,7 @@ class MoreInfoLight extends LocalizeMixin(EventsMixin(PolymerElement)) {
     if (stateObj && stateObj.state === "unavailable") {
       classes.push("is-unavailable");
     }
+    classes.push("content");
```

Comment:
You could add this to line 295 when the list is initialized. Also really don't care as this code will go away 🤷

Reply:
Curious now, plans for more-info overhaul?

Reply:
All Polymer and JavaAcript will go away, eventually...

Reply:
Oh sure, I thought it was something more :)